pointerevents/pointerevent_attributes_mouse-manual.html

```diff
@@ -17,68 +17,87 @@
             // will fail unless the async_test is created with the var name "test_pointerEvent".
             add_completion_callback(showPointerTypes);
 
+            function checkPointerEventAttributes(event, targetBoundingClientRect, testNamePrefix, pointerEventInterface) {
+                if (detected_eventTypes[event.type])
+                    return;
+                var pointerTestName = testNamePrefix + ' ' + event.pointerType + ' ' + event.type;
+                test(function () {
+                    assert_equals(event.pointerType, 'mouse', 'pointerType should be mouse');
+                }, pointerTestName + ".pointerType attribute is correct.");
+
+                detected_pointertypes[event.pointerType] = true;
+                // Test button and buttons
+                if (event.type == 'pointerdown') {
+                    test(function() {
+                        assert_true(event.button == 0, "If left mouse button is pressed button attribute is 0")
+                    }, pointerTestName + "'s button attribute is 0 when left mouse button is pressed.");
+                    test(function() {
+                        assert_true(event.buttons == 1, "If left mouse button is pressed buttons attribute is 1")
+                    }, pointerTestName + "'s buttons attribute is 1 when left mouse button is pressed.");
+                } else if (event.type == 'pointerup') {
+                    test(function() {
+                        assert_true(event.button == 0, "If left mouse button is just released button attribute is 0")
+                    }, pointerTestName + "'s button attribute is 0 when left mouse button is just released.");
+                    test(function() {
+                        assert_true(event.buttons == 0, "If left mouse button is just released buttons attribute is 0")
+                    }, pointerTestName + "'s buttons attribute is 0 when left mouse button is just released.");
+                } else {
+                    test(function() {
```

Comment:
I updated the test descriptions. I hope this one looks better.
Regarding merging button and buttons tests I believe it is better to keep them separate. I see no problem with having a lot of small tests. As long as user doesn't need to do separate stuff for them. Particularly button and buttons seem different enough to have different tests for them.